k2/csrc/array_ops.cu

```diff
@@ -147,63 +148,55 @@ Array1<int32_t> SpliceRowSplits(int32_t num_arrays,
 
 bool ValidateRowIds(Array1<int32_t> &row_ids, Array1<int32_t> *temp) {
```

Comment:
I would suggest giving a better name for `temp`, e.g.,  `invalid`.

Reply:
The result (valid or not) has been returned by the function, tmp is just used to save one allocation.